test/acceptance/features/steps/openshift.py

```diff
@@ -334,3 +335,25 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
             return lst
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
+
+    def get_secret_data(self, secret_name, secret_key, namespace, secret_value):
```

Comment:
the function should return value under the given key. Hence, we should remove argument `secret_value` from the method signature.

Reply:
So should I move the secret_value related logic under the step directly?

Reply:
Up to you - if there are no two steps sharing the logic, it could me moved into the step function.

Reply:
Shifted functionality to step, so that only openshift specific functions reside in openshift.py

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/597/commits/f0136cc8565b4aa248f5dafc58922680e88edd03